proto/exocore/delegation/v1/query.proto

```diff
@@ -17,43 +17,45 @@ message DelegationInfoReq {
   string asset_id = 2 [(gogoproto.customname) = "AssetID"];
 }
 
+// StakerMap is a map to save a batch of stakers
+message StakerMap {
+  // stakers is a map, the bool value is useless, just because the protobuf
+  // doesn't support the nil value. The key is stakerID
+  map<string,bool> stakers = 1;
```

Comment:
Why not repeated string?

Reply:
The list needs to consider the repeated item, so use a map.